go/vt/discovery/keyspace_events.go

```diff
@@ -134,6 +153,7 @@ func (kss *keyspaceState) beingResharded(currentShard string) bool {
 	// for all the known shards, try to find a primary shard besides the one we're trying to access
 	// and which is currently healthy. if there are other healthy primaries in the keyspace, it means
 	// we're in the middle of a resharding operation
+	// FIXME: probably doesn't work for anything other than 1->2 resharding
```

Comment:
This sounds like a big limitation. No? Sounds like it's not something new though. :-)

Reply:
my understanding is m -> n split works when m < n